modules/insights-operator-new-pull-secret-enable.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * support/insights-operator/enabling-remote-health-reporting.adoc
+
+[id="insights-operator-new-pull-secret-enable_{context}"]
+= Modifying your global cluster pull secret to enable remote health reporting
+
+You can modify your existing global cluster pull secret to enable remote health reporting. If you have previously disabled remote health monitoring, you must first download a new pull secret with your `cloud.openshift.com` access token from Red Hat OpenShift Cluster Manager. 
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to Red Hat OpenShift Cluster Manager.
```

Comment:
I would recommend to avoid the redundancy that you revise this as:
"Access to:
- cluster as user...
- Red Hat OpenShift...